docs/animation-format.markdown

```diff
@@ -58,40 +58,39 @@ See below for a fully valid example Texture Atlas, annotated with information on
-	<SubTexture name="Forward0003" x="39" y="368" width="39" height="44"/>
+  <SubTexture name="Forward0002" x="39" y="368" width="39" height="44"/>
+  <SubTexture name="Forward0003" x="39" y="368" width="39" height="44"/>
 
   <!--
     Fun fact, an easy way to create an animation which is the reverse or remixing of another one is to
     copy the original animation, rearrange the frames as desired, then change the frame numbers
     such that they are in order, like so:
   -->
-	<SubTexture name="Backward0000" x="39" y="368" width="39" height="44"/>
-	<SubTexture name="Backward0001" x="39" y="368" width="39" height="44"/>
+  <SubTexture name="Backward0000" x="39" y="368" width="39" height="44"/>
+  <SubTexture name="Backward0001" x="39" y="368" width="39" height="44"/>
   <SubTexture name="Backward0002" x="0" y="368" width="39" height="44"/>
   <SubTexture name="Backward0003" x="0" y="368" width="39" height="44"/>
 
   <!--
     Sub-textures can partially or fully overlap each other without conflicts.
   -->
-	<SubTexture name="Testing" x="50" y="50" width="100" height="100" />
-	<SubTexture name="MoreTesting" x="75" y="75" width="25" height="25"/>
+  <SubTexture name="Testing" x="50" y="50" width="100" height="100" />
+  <SubTexture name="MoreTesting" x="75" y="75" width="25" height="25"/>
 
   <!--
-    HaxeFlixel supports many additional attributes on subtextures, that grant a fine degree of control
-    over how an individual frame appears. When used properly, this can provide massive memory savings,
-    since having a large graphic makes your game use more memory than, and therefore run slower than,
-    a smaller graphic.
+    HaxeFlixel supports many additional attributes on subtextures that grant a fine degree of control
+    over how an individual frame appears. When used properly, this can save massive amounts of memory,
+    since having a large graphic makes your game use more memory than a smaller graphic.
   -->
 
   <!--
-    The frame attributes allow you to define a larger frame which your subtexture appears in.
+    A frame's attributes allow you to define a larger frame in which your subtexture appears.
 
-    Think of the example of an explosion animation. At certain parts of the animation, parts of the texture
+    Think of an explosion animation. At certain parts of the animation, parts of the texture
     will be completely empty, but are necessary to ensure the explosion will animate properly.
 
-    With the frame attributes, you cut out the transparent parts completely when generating your texture,
-    then readd them by defining the full size of the final image, as well as where in your image the texture should be positioned.
+    With frame attributes, you cut out the transparent parts completely when generating your texture,
+    then read them by defining the full size of the final image, as well as where in your image the texture should be positioned.
```

Comment:
This is supposed to be readd (re-add?). The transparent parts had been previously removed, so now you are adding them back in.

- [ ] Think of a new wording for this?